Source/embeddings/Builder/RemoveDecoratedMethod.ts

```diff
@@ -2,22 +2,22 @@
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
 import { Constructor } from '@dolittle/types';
-import { EmbeddingClassRemoveMethod } from './EmbeddingClassRemoveMethod';
+import { EmbeddingClassDeletionMethod } from './EmbeddingClassDeletionMethod';
 
 /**
- * Represents methods decorated with the @remove() decorator.
+ * Represents methods decorated with the @resolveDeletionToEvents() decorator.
  */
 export class RemoveDecoratedMethod {
```

Comment:
```suggestion
export class DeletionDecoratedMethod {
```
Also filename rename